ComponentKit/Utilities/CKComponentActionInternal.h

```diff
@@ -57,11 +61,11 @@ class CKTypedComponentActionBase {
 
   bool operator==(const CKTypedComponentActionBase& rhs) const;
 
-private:
   // Destroying this field calls objc_destroyWeak. Since this is the only field
   // that runs code on destruction, making this field the first field of this
   // object saves an offset calculation instruction in the destructor.
   __weak id _targetOrScopeHandle;
+  dispatch_block_t _block;
```

Comment:
If binary bloat of this is too high, then we could consider taking the scope handle/target and moving it *into* the block? That may have other side-effects though, and I'm not sure exactly what would happen to our demotion/promotion callsites.